ca_tools/rviz/multi_robot.rviz

```diff
@@ -273,7 +273,7 @@ Visualization Manager:
     - Class: rviz/SetInitialPose
       Topic: /initialpose
     - Class: rviz/SetGoal
-      Topic: /move_base_simple/goal
+      Topic: /create1/move_base_simple/goal
```

Comment:
There is another topic that you could edit in this config file: `/initialpose`.

Reply:
Adressed